core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -60,7 +60,8 @@ class ExecutorSummary private[spark](
     val totalShuffleRead: Long,
     val totalShuffleWrite: Long,
     val maxMemory: Long,
-    val executorLogs: Map[String, String])
+    val executorLogs: Map[String, String],
+    val isRemoved: Boolean)
```

Comment:
Mind adding a `//` comment next to `isRemoved` to say something like "Added in 1.5.0" or "since 1.5.0"?